### PR TITLE
Navbar And HomeScreen Completed

### PR DESCRIPTION
**Pull request by m26unkwn**
1. Worked on Responsiveness of the website

### CODE REVIEW

File: src/screens/navbar/sidebar/sidebar.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/** @format */
+
+.sidebar-wrapper {
+  grid-area: sidebar;
+  width: 15rem;
+  height: 100vh;
+  background-color: white;
```
Comment (deekshithmd) [inline src/screens/navbar/sidebar/sidebar.css:7]: suggestion: better to use color variable

File: src/screens/navbar/sidebar/sidebar.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+.sidbar-nav-items > * {
+  margin-top: 1rem;
+}
+
+.sidebar-items {
+  display: flex;
+  align-items: center;
+  gap: 2rem;
+  color: black;
```
Comment (deekshithmd) [inline src/screens/navbar/sidebar/sidebar.css:37]: same here, using color variable will be good

**Review by deekshithmd — APPROVED** [review]

File: src/components/Cards/card.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
+/**
+ * /* category cards
+ *
+ * @format
+ */
+
+.category-item {
+  width: 200px;
+  height: 250px;
```
Comment (dev-ashishsethi) [inline src/components/Cards/card.css:9]: use rems here

File: src/screens/home/hero/hero.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+.hero-action > button {
+  color: black;
+  font-family: "poppins";
+  font-weight: 500;
+  background: var(--bg-color);
+  border-radius: 4px;
+  width: 12rem;
+}
+button > img {
+  margin-left: 0.5rem;
```
Comment (dev-ashishsethi) [inline src/screens/home/hero/hero.css:33]: for size less than 1 rem use pixels

**Review by dev-ashishsethi — APPROVED** [review]
do check comments

File: src/screens/home/hero/hero.css [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  background: linear-gradient(
+    90deg,
+    rgba(2, 0, 36, 1) 0%,
+    rgba(52, 52, 255, 1) 0%,
+    rgba(0, 212, 255, 1) 100%
+  );
+  color: var(--text-secondary-color);
+}
+.hero-title-head {
+  font-size: clamp(2rem, 4vw, 5rem);
```
Comment (saurabhchirde) [inline src/screens/home/hero/hero.css:15]: This is something new for me.

**Review by saurabhchirde — APPROVED** [review]
Liked your Home page. It's clean and attractive. 
Written code is clean and easy to read. You can proceed to merge.

File: src/screens/navbar/sidebar/sidebar.css (hunk 1 above)
Comment (m26unkwn) [reply src/screens/navbar/sidebar/sidebar.css:7]: ok will add the varaible

File: src/screens/home/hero/hero.css (hunk 5 above)
Comment (m26unkwn) [reply src/screens/home/hero/hero.css:15]: thanks

**Pull request merged**